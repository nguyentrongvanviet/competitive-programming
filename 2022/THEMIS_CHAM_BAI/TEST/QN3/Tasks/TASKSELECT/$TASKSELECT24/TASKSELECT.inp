10
3 564581648
2 328407516
3 448628987
3 571947891
3 826208185
3 26513306
5 762214245
2 715585004
3 197984872
5 1000000000
