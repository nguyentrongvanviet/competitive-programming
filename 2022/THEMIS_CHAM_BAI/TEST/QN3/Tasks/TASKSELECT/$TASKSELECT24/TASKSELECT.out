636636588
164203757
322999711
358149157
268783788
602587431
783793485
357792501
673213430
603125018
